fe1-web/src/features/rollCall/reducer/RollCallReducer.ts

```diff
@@ -199,7 +199,7 @@ export const getRollCallById = (rollCallId: Hash | string, state: unknown) => {
  *
  * @param rollCallId - The id of the roll call
  */
-export const makeRollCallAttendeesListSelector = (rollCallId: Hash | string | undefined) => {
+export const makeRollCallAttendeesSelector = (rollCallId: Hash | string | undefined) => {
```

Comment:
considering there's no obvious return type from this selector definition, is it a good idea to remove the `List` descriptor from the name? What do you think? I'm not 100% convinced.

Reply:
I think you're right, I'll rename if back to what it was.